# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1261294529 28800
#      Sat Dec 19 23:35:29 2009 -0800
# Node ID 061ea41a3da7235e5db79169cdf0c63b8edebaad
# Parent  b250fee61632acbf577ba92951c15f4a0b2af9a7
Reimplement previous patch with distribution triggered by m4/module.mk for clarity

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2009-12-19  Rik <octave@nomad.inbox5.com>
+
+	* m4/module.mk, Makefile.am: Redo previous patch with distribution
+	triggered by m4/module.mk.
+
 2009-12-19  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Distribute m4/gnulib-cache.m4 per recommendation
 
 2009-12-19  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am, examples/Makefile.am, common.mk, configure.ac: Eliminate
 	TOPDIR variable in favor of built-in automake variables of top_builddir
diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -47,17 +47,16 @@ EXTRA_DIST = \
   README.Cygwin \
   README.Linux \
   README.MSVC \
   README.Windows \
   README.kpathsea \
   ROADMAP \
   SENDING-PATCHES \
   autogen.sh \
-  m4/gnulib-cache.m4 \
   missing \
   mk-opts.pl \
   mkinstalldirs \
   mkoctfile.cc.in \
   mkoctfile.in \
   move-if-change \
   octave-bug.cc.in \
   octave-bug.in \
diff --git a/m4/module.mk b/m4/module.mk
--- a/m4/module.mk
+++ b/m4/module.mk
@@ -1,11 +1,12 @@
 EXTRA_DIST += \
   m4/module.mk \
   m4/acx_blas.m4 \
   m4/acx_blas_f77_func.m4 \
   m4/acx_lapack.m4	\
   m4/acx_pthread.m4 \
+  m4/gnulib-cache.m4 \
   m4/libtool.m4 \
   m4/ltoptions.m4 \
   m4/ltsugar.m4 \
   m4/ltversion.m4 \
   m4/lt~obsolete.m4
